src/System.Windows.Forms/src/System/Windows/Forms/ListBox.AccessibleObject.cs

```diff
@@ -251,43 +281,52 @@ internal override void SetFocus()
 
             if (!_itemAccessibleObjects.ContainsKey(item))
             {
-                _itemAccessibleObjects.Add(item, CreateItemAccessibleObject(_owningListBox, item));
+                _itemAccessibleObjects.Add(item, CreateItemAccessibleObject(owner, item));
             }
 
             return _itemAccessibleObjects[item];
         }
 
         public override int GetChildCount()
         {
-            return _owningListBox.Items.Count;
+            return this.TryGetOwnerAs(out ListBox? owner) ? owner.Items.Count : 0;
         }
 
         public override AccessibleObject? GetFocused()
         {
-            int index = _owningListBox.FocusedIndex;
-            if (index >= 0)
+            this.TryGetOwnerAs(out ListBox? owner);
+            if (owner is not null)
```

Comment:
Nit:
```suggestion
            if (this.TryGetOwnerAs(out ListBox? owner))
```
`owner` parameter has `[NotNullWhen(true)]` attribute, so when it returns true, it's guaranteed to be not null. Same  can be applied in `GetSelected`